services/masterdata/src/lib/components/masterdatas-list/masterdatas-list.component.html

```diff
@@ -5,21 +5,35 @@ <h3>
 				{{ masterdataType?.name }}
 				<small>({{ masterdataType?.masterdatasCount }})</small>
 			</h3>
-			<lens-button
-				label="Add masterdata"
-				icon="pi pi-plus"
-				type="success"
-				[disabled]="isLoading"
-				(clicked)="onAddAction()">
-      </lens-button>
+      <div class="masterdata-table-toolbar p-fluid">
+        <div class="masterdata-table-toolbar-item">
+          <lens-button
+            label="Add masterdata"
+            icon="pi pi-plus"
+            type="success"
+            [disabled]="isLoading"
+            (clicked)="onAddAction()"></lens-button>
+        </div>
+        <div class="masterdata-table-toolbar-item">
+          <lens-services-tags-selector
```

Comment:
Is lens-services-tags-selector the correct name? Shouldn't it be lens-tags-selector?

Reply:
I will let it like this for now. I have another selector with the same prefix in masterdata: lens-services-masterdata-type-selector

Reply:
I can't find `lens-services-tags-selector`. Where is it declared?

Reply:
@gitout0fhere /libs/framework/services/masterdata/src/lib/core/ui/tags-selector